srcpkgs/qutebrowser/template

```diff
@@ -21,7 +21,10 @@ build_options="webengine"
 desc_option_webengine="Build Qt5 WebEngine support"
 
 if [ "$XBPS_TARGET_ENDIAN" = "le" ]; then
-	if [ "$XBPS_TARGET_WORDSIZE" = "$XBPS_WORDSIZE" ]; then
+	# qt5-webengine is only available for little-endian systems
+	if [ "$XBPS_TARGET_WORDSIZE" = "$XBPS_WORDSIZE" -a -z "$CROSS_BUILD" ]; then
```

Comment:
Native build imply wordsize equality.

Reply:
Good point, thanks. I also forgot to remote `broken` for `aarch64*` in `python-PyQt5-webengine`, which is not needed with `nocross`. Both are fixed.